.github/workflows/test.yml

```diff
@@ -495,6 +495,7 @@ jobs:
       OVN_SECOND_BRIDGE: "${{ matrix.second-bridge == '2br' }}"
       ENABLE_MULTI_NET: "${{ matrix.target == 'multi-homing' || matrix.target == 'kv-live-migration' || matrix.target == 'network-segmentation' || matrix.target == 'tools' || matrix.target == 'multi-homing-helm' || matrix.target == 'traffic-flow-test-only' || matrix.routeadvertisements != '' }}"
       ENABLE_NETWORK_SEGMENTATION: "${{ matrix.target == 'network-segmentation' || matrix.network-segmentation == 'enable-network-segmentation' }}"
+      ENABLE_PRE_CONF_UDN_ADDR: "${{ matrix.preconfigured-udn-addresses == 'true' }}"
```

Comment:
is it work it to have a new matrix knob ? maybe we can just configure depending on the others

Reply:
Good point, done.